index.d.ts

```diff
@@ -1,10 +1,11 @@
 import {Options as FastGlobOptions} from 'fast-glob';
+import {Stats} from 'fs';
 
 declare namespace globby {
 	type ExpandDirectoriesOption =
 		| boolean
 		| readonly string[]
-		| {files?: readonly string[]; extensions?: readonly string[]};
+		| { files?: readonly string[]; extensions?: readonly string[] };
```

Comment:
Don't do unrelated changes.

Reply:
Sorry, this is probably auto format. You don't seem to provide a prettier config or eslintrc ?

Reply:
> You don't seem to provide a prettier config or eslintrc ?

No, but you should default to not formatting if such configs do not exist.